lib/Elastica/Aggregation/DateHistogram.php

```diff
@@ -143,14 +143,16 @@ public function setFormat($format)
      */
     public function setExtendedBounds($min = '', $max = '')
     {
-        $bounds = array();
-        $bounds['min'] = $min;
-        $bounds['max'] = $max;
         // switch if min is higher then max
-        if (strtotime($min) > strtotime($max)) {
-            $bounds['min'] = $max;
-            $bounds['max'] = $min;
-        }
+        $bounds = strtotime($min) > strtotime($max)
```

Comment:
The key for me in writing code is to make it as readable as possible. I'm not sure if this change makes the code more readable (at least not for me).